modules/ROOT/pages/about-my-applications.adoc

```diff
@@ -12,14 +12,28 @@ image::ex2-my-applications.png[]
 A contract holds information about the contract with a client ID and client secret for the API.
 Each application lists a description, its URL, redirect URIs, grant types, and a usage dashboard.
 
-The usage information lets you set the verification duration. If not specified, the range is the last 8 days. The usage lists the total number of requests to the API, average latency, the error rate, and graphs for total requests, average latency, requests by HTTP status code, and the error percentage.
+The usage information lets you set the verification duration. If not specified,
+the range is the last 8 days. The usage lists the total number of requests to
+the API, average latency, the error rate, and graphs for total requests,
+average latency, requests by HTTP status code, and the error percentage.
 
 The applications result when users click Request Access to establish
```

Comment:
applications result? The application is created? I don't know what "applications result" means. 
If you don't have an application to link to your API, you can create the application by clicking the *Request Access* menu. You can then click *Request Access* to establish a link between your application and API.

Reply:
That's my best guess. I'm not sure what's going on there.

Reply:
I rewrote for clarity, and included this text. There was a lot of sloppy text in this one, including things I missed such as "average latency" being repeated. Thank you for a very helpful review!